.github/workflows/pr-code-coverage.yml

```diff
@@ -46,6 +46,10 @@ jobs:
 
       - name: Install and Build
         uses: ./.github/actions/install-build
+
+      - name: Install composer dev dependencies
+        working-directory: plugins/woocommerce
+        run: composer install
```

Comment:
@ObliviousHarmony I added this extra command to some of the workflows that relied on composer dev dependencies like `phpunit` and `phpcs`.
Apparently this worked prior because of the use of `install-if-deps-outdated`, which in turn would install all the dev dependencies anyway.
Instead of making this change in `install-build` I added it as a extra task, given we don't need to install the dev packages for some of the other workflows.

Reply:
I actually end up updating the `install-build` script to allow for the `composer_no_dev` input to be passed in. Default to 1.